helm/chaos-mesh/README.md

```diff
@@ -29,9 +29,10 @@ The following tables list the configurable parameters of the Chaos Mesh chart an
 | `imagePullSecrets` | Global Docker registry secret names as an array | [] (does not add image pull secrets to deployed pods) |
 | `controllerManager.securityContext` | Pod securityContext if needed | `{}` |
 | `controllerManager.hostNetwork` | Running chaos-controller-manager on host network | `false` |
-| `controllerManager.allowHostNetworkTesting` | Allow testing on `hostNetwork` pods | `false` |
-| `controllerManager.serviceAccount` | The serviceAccount for chaos-controller-manager | `chaos-controller-manager` |
-| `controllerManager.serviceAccountAnnotations` | ServiceAccount annotations for chaos-controller-manager | `{}` |
+| `controllerManager.allowHostNetworkTesting`   | Allow testing on `hostNetwork` pods | `false` |
+| `controllerManager.serviceAccount.create` | Create the serviceAccount for chaos-controller-manager | `true` |
+| `controllerManager.serviceAccount.name` | The serviceAccount name for chaos-controller-manager | `chaos-controller-manager` |
+| `controllerManager.serviceAccount.annotations` | ServiceAccount annotations for chaos-controller-manager | `{}` |
```

Comment:
if you could keep the original fields name(changes like `serviceAccount` -> `servceAccount.name`,`serviceAccountAnnotations`->`serviceAccount.annotations` ), do not introduce breaking changes, we would carry this PR in the next nearly release. Or we could not carry it in the next minor release.

Reply:
@STRRL - Thank you for your feedback, I reverted what I changed in order to not break anything. Please re-review when you have time. Thank you!